include/commissioner/commissioner.hpp

```diff
@@ -1133,6 +1133,15 @@ class Commissioner
      * @param[in]      aJoinerId      A Joiner ID.
      */
     static void AddJoiner(ByteArray &aSteeringData, const ByteArray &aJoinerId);
+
+    /**
+     * @brief Return the commissioner version.
+     *
+     * @return A version string in format of <MAJOR>.<MINOR>.<PATCH>[-<GIT_REVISION>].
```

Comment:
```suggestion
     * @return A version string in format of <MAJOR>.<MINOR>.<PATCH>[+<GIT_REVISION>].
```
inline with https://semver.org/#spec-item-10